user-guide/Troubleshooting/Procedures/Investigating_dataminerservices_feature_issues.md

```diff
@@ -28,6 +28,9 @@ You can validate this as follows:
 
    This way, you avoid dealing with issues that have already been fixed.
 
+> [!NOTE]
+> You can always install the [latest Cloud Pack](https://aka.dataminer.services/cloud-pack-download) manually to upgrade or install the core DxMs required for upgrades via the Admin app should there be DxMs that are missing for the remote upgrades to work.
```

Comment:
To make this a bit easier to read, I'd rephrase it as follows:
```suggestion
> If any of the DxMs are missing that are required for the upgrade from the Admin app to work, download and install the [latest Cloud Pack](https://aka.dataminer.services/cloud-pack-download) to install these manually.
```

Reply:
Also good for me.